content-api/content-service/app/controllers/BaseController.scala

```diff
@@ -199,6 +199,46 @@ abstract class BaseController(protected val cc: ControllerComponents)(implicit e
 
     def validateContentType(input: java.util.Map[String, AnyRef]): Boolean = StringUtils.isNotBlank(input.getOrDefault("contentType", "").asInstanceOf[String])
 
+    /**
+     * Validates whether the end date is greater than start date.
+     *
+     * @param input the Event Body.
+     * @return true : If endDate is > startDate; false : otherwise.
+     */
+    def validateDates(input: java.util.Map[String, AnyRef]): Boolean = {
```

Comment:
Time also needs to be factored in for comparison, not just date. 

Eg: it should be possible to create an event starting 11am and ending 5pm on the same day

Reply:
Time is also considered now while comparison. I have pushed the new code.